docs/book/src/getting-started.md

```diff
@@ -1,3 +1,645 @@
 # Getting Started
 
-{{#embed-github repo:"kubernetes-sigs/cluster-api" path:"docs/book/src/user/quick-start.md"}}
+In this tutorial we'll cover the basics of how to use Cluster API Provider AWS to create one or more Kubernetes clusters.
+
+## Installation
+
+### Common Prerequisites
+
+- Install and setup [kubectl] in your local environment
+- Install [kind] and [Docker]
+
+### Install and/or configure a Kubernetes cluster
+
+Cluster API requires an existing Kubernetes cluster accessible via kubectl. During the installation process the
+Kubernetes cluster will be transformed into a [management cluster] by installing the Cluster API [provider components], so it
+is recommended to keep it separated from any application workload.
+
+It is a common practice to create a temporary, local bootstrap cluster which is then used to provision
+a target [management cluster] on the selected [infrastructure provider].
+
+**Choose one of the options below:**
+
+1. **Existing Management Cluster**
+
+   For production use-cases a "real" Kubernetes cluster should be used with appropriate backup and disaster recovery policies and procedures in place. The Kubernetes cluster must be at least v1.20.0.
+
+   ```bash
+   export KUBECONFIG=<...>
+   ```
+**OR**
+
+2. **Kind**
+
+   <aside class="note warning">
+
+   <h1>Warning</h1>
+
+   [kind] is not designed for production use.
+
+   **Minimum [kind] supported version**: v0.17.0
+
+   **Help with common issues can be found in the [Troubleshooting Guide](./topics/troubleshooting.md).**
+
+   </aside>
+
+   [kind] can be used for creating a local Kubernetes cluster for development environments or for
+   the creation of a temporary [bootstrap cluster] used to provision a target [management cluster] on the selected infrastructure provider.
+
+   The installation procedure depends on the version of kind; if you are planning to use the Docker infrastructure provider,
+   please follow the additional instructions in the dedicated tab:
+
+   {{#tabs name:"install-kind" tabs:"Default,Docker,KubeVirt"}}
+   {{#tab Default}}
+
+   Create the kind cluster:
+   ```bash
+   kind create cluster
+   ```
+   Test to ensure the local kind cluster is ready:
+   ```bash
+   kubectl cluster-info
+   ```
+
+   Then follow the instruction for your kind version using  `kind create cluster --config kind-cluster-with-extramounts.yaml`
+   to create the management cluster using the above file.
+
+   {{#/tab }}
```

Comment:
Anything related to /tab tags are not resolving correctly. Please check in the doc preview and correct it.

Reply:
Actually in markdown preview /tab will not render correctly. But it will work fine in the Book because we render it in .html format.